spec/features/saml/loa1_sso_spec.rb

```diff
@@ -18,6 +18,19 @@
 
       expect(current_url).to eq @saml_authn_request
     end
+
+    it 'takes user to agency handoff page when sign in flow complete' do
```

Comment:
Rename this test to reflect the two expectations?
1) the user is redirected to the SP after fully authenticating 
2) the user can then access the IdP directly without being redirected back to the SP

By keeping these 2 expectations here, we can remove the scenarios that start on lines 7 and 23 in `spec/features/saml/test_sp_spec.rb` since those are already tested here.